corgi/core/fixups.py

```diff
@@ -261,73 +87,25 @@
-        "cpe:/a:redhat:satellite:6.11::el8",
-        "cpe:/a:redhat:rhel_satellite_client:6::el9",
-        "cpe:/a:redhat:satellite_maintenance:6.11::el8",
-        "cpe:/a:redhat:satellite_capsule:6.11::el8",
-        "cpe:/a:redhat:rhel_satellite_client:6::el6",
-        "cpe:/a:redhat:rhel_satellite_client:6::el7",
-    ],
-    "rhn_satellite_6.12": [
-        "cpe:/a:redhat:satellite_utils:6.12::el8",
-        "cpe:/a:redhat:satellite_maintenance:6.12::el8",
-        "cpe:/a:redhat:satellite:6.12::el8",
-        "cpe:/a:redhat:satellite_capsule:6.12::el8",
-    ],
-    "rhn_satellite_6.13": [
-        "cpe:/a:redhat:satellite:6.13::el8",
-        "cpe:/a:redhat:satellite_capsule:6.13::el8",
-        "cpe:/a:redhat:satellite_maintenance:6.13::el8",
-        "cpe:/a:redhat:satellite_utils:6.13::el8",
-    ],
-    "rhods-1.20": ["cpe:/a:redhat:openshift_data_science:1.20::el8"],
-    "rhods-1.21": ["cpe:/a:redhat:openshift_data_science:1.21::el8"],
-    "rhods-1.23": ["cpe:/a:redhat:openshift_data_science:1.23::el8"],
-    "rhods-1.24": ["cpe:/a:redhat:openshift_data_science:1.24::el8"],
+    # None of the matching variants are live
     "rhosc-1": ["cpe:/a:redhat:openshift_sandboxed_containers:1.3.0::el8"],
-    "rhos_devspaces-3": ["cpe:/a:redhat:openshift_devspaces:3::el8"],
-    "rhosdt-2": ["cpe:/a:redhat:openshift_distributed_tracing:2.3::el8"],
+    # Not getting match with pattern matching, excluded from brew_tag matching because it shadows
+    # dts-11.0.z
     "rhscl-3.8.z": [
         "cpe:/a:redhat:rhel_software_collections:3::el6",
         "cpe:/a:redhat:rhel_software_collections:3::el7",
     ],
-    "rhui-4": ["cpe:/a:redhat:rhui:4::el8"],
-    "rodoo-1": ["cpe:/a:redhat:run_once_duration_override_operator:1.0::el8"],
-    "rosa-cli-1": ["cpe:/a:redhat:openshift_service_on_aws:1::el8"],
-    "snr-0.4.z": ["cpe:/a:redhat:workload_availability_self_node_remediation:0.4::el8"],
-    "srvcom-1": ["cpe:/a:redhat:serverless:1.0::el8", "cpe:/a:redhat:serverless:1.25::el8"],
-    "stf-1.4": ["cpe:/a:redhat:service_telemetry_framework:1.4::el8"],
-    "stf-1.5": ["cpe:/a:redhat:service_telemetry_framework:1.5::el8"],
-    "wto-1.5": ["cpe:/a:redhat:webterminal:1.5::el8"],
-    "rhcertification-6": ["cpe:/a:redhat:certifications:1::el6"],
-    "rhcertification-7": ["cpe:/a:redhat:certifications:1::el7"],
-    "rhcertification-8": ["cpe:/a:redhat:certifications:1::el8"],
-    "rhcertification-9": ["cpe:/a:redhat:certifications:1::el9"],
 }
 
 
-def cpe_lookup(product_stream_name: str) -> list[str]:
-    """Temporary cpe fixup"""
-    return supported_stream_cpes.get(product_stream_name, [])
+def cpe_lookup(product_stream_name: str) -> Optional[set[str]]:
+    """Manual CPE overrides for streams which cannot be matched with variants"""
+    if product_stream_name in supported_stream_cpes:
+        return set(supported_stream_cpes.get(product_stream_name, []))
+    else:
+        return set()
```

Comment:
You don't need to do .get() here because you've already checked that this stream_name is in the dict of supported_streams. You can do a one-liner that matches the old code:
```suggestion
    return set(supported_stream_cpes.get(product_stream_name, []))
```
This will aways return either a set of the CPEs, or an empty set. Or you can do an if / else without .get():
```suggestion
    if product_stream_name in supported_stream_cpes:
        return set(supported_stream_cpes[product_stream_name])
    else:
        return set()
```

Reply:
Using the first suggestion, thanks.